internal/proxy/task_search.go

```diff
@@ -490,38 +492,38 @@ func (t *searchTask) initSearchRequest(ctx context.Context) error {
 	return nil
 }
 
-func (t *searchTask) tryGeneratePlan(params []*commonpb.KeyValuePair, dsl string) (*planpb.PlanNode, *planpb.QueryInfo, int64, error) {
+func (t *searchTask) tryGeneratePlan(params []*commonpb.KeyValuePair, dsl string) (*planpb.PlanNode, *planpb.QueryInfo, *schemapb.FieldSchema, int64, error) {
```

Comment:
how about adding fieldID to planpb.QueryInfo?

Reply:
Seems we don't transfer QueryInfo to segcore or delegator.